CHANGES.md

```diff
@@ -15,6 +15,16 @@
 [Unreleased](https://github.com/bird-house/birdhouse-deploy/tree/master) (latest)
 ------------------------------------------------------------------------------------------------------------------
 
+## Fixes
+
+- Wrong compose up extra arguments given to compose restart
+
+  * For example, when setting `BIRDHOUSE_COMPOSE_UP_EXTRA_OPTS="--remove-orphans"` in
+    `env.local`, that `--remove-orphans` flag is not supposed to be used with
+    compose restart.  Fix regression from
```

Comment:
just use the `code` for the `up` and `restart` command

Reply:
done https://github.com/bird-house/birdhouse-deploy/pull/515/commits/e6b264712375bf755bdb9ec0af2bc3aea1c841d0